notebook-examples/kge_wikimedia.ipynb

```diff
@@ -0,0 +1,1396 @@
+{
```

Comment:
<p>I think we should deal with this Error output. And use build from source example to make dglke_score work.</p><p> </p><p>Others looks good to me. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/dgl-ke/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>@classicsong</p><p> </p><p>I have updated the notebook per feedback.</p><p> </p><p>Please take a look.</p>